content/v2/overview/authentication.md

```diff
@@ -5,18 +5,24 @@ title: API v2 - Overview - Authentication
 # Authentication
 
 SeeClickFix currently supports the following authentication methods for access to API v2.
-1. HTTP Basic Authentication
+1. Token Authentication
 1. OAauth2 Authentication
 
-### HTTP Basic Authentication
+### Token Authentication
+This method is appropriate for API access. It is a secret that is included as a request header.
```

Comment:
Thoughts on linking out to additional information from https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#bearer ?

Reply:
Personally I felt there was not any helpful information in that for someone who just wants to make something work with a bearer token. It's a helpful overview of http auth, but I don't think it's helpful for the next level of understanding about bearer tokens.